src/main/java/com/dev/cinema/controllers/CinemaHallController.java

```diff
@@ -0,0 +1,41 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.dto.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.CinemaHallResponseDto;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.util.CinemaHallConvertUtil;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallConvertUtil cinemaHallConvertUtil;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallConvertUtil cinemaHallConvertUtil) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallConvertUtil = cinemaHallConvertUtil;
+    }
+
+    @PostMapping("/cinema-halls")
```

Comment:
```suggestion
    @PostMapping("/cinema-halls")
```
винеси в загальний RequestMapping над класом

Reply:
done